src/components/Header.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { ChangeEvent, FormEvent } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  newTodoTitle: string,
+  setNewTodoTitle: (title: string) => void,
+  addTodo: (event: FormEvent<HTMLFormElement>) => void,
+  isLoading: boolean,
+  toggleAllcompletedTodos: () => void,
```

Comment:
Rename 'toggleAllcompletedTodos' to 'toggleAllCompletedTodos' to maintain consistent camelCase naming convention.